.github/workflows/build-package-docs.yaml

```diff
@@ -120,3 +131,94 @@ jobs:
         name: package-docs-build
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
+
+  deploy-package-docs:
+    if: fromJSON(github.event.inputs.deploy)
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    environment:
+      name: deploy-package-docs
+      url: ${{ env.ENV_URL }}
```

Comment:
I have no idea whether that works. I would assume that GHA expects this value to be there when the job is started, and won't allow it to change during the job. I have no idea what `url` actually does though, so :shrug:

Reply:
@felixfontein I tested this on my fork and it seems to work (if I understand correctly what the `url` is doing).

Here are a couple of my test runs:
- https://github.com/oraNod/ansible-documentation/actions/runs/9415222788
- https://github.com/oraNod/ansible-documentation/actions/runs/9415206997

On the summary page for the run, you can see the url is populated as part of the deploy job:

![image](https://github.com/ansible/ansible-documentation/assets/32749632/94f7f99d-9366-41d2-9fba-cdd64af04f39)

![image](https://github.com/ansible/ansible-documentation/assets/32749632/9d0d0e33-8899-4577-a49b-dc8a759e22f4)

Reply:
Ah, so this is "only" a URL that's shown in the UI. In that case it makes sense :)